PostInstallCob4.sh

```diff
@@ -131,25 +133,20 @@ function  SynchronizeRobotUser {
 function SetupRobotBashrcWorkspace {
 
   echo -e "\n${green}INFO:Setup Robot Bashrc Workspace${NC}\n"
-  Entry
+
+  source /opt/ros/indigo/setup.bash #FIXME only working for indigo!!!
 
   if grep -q ROBOT "/u/robot/.bashrc"; then  
     echo ".bashrc already configured"
   else
-    cp /u/robot/git/setup_cob4/cob-pcs/user.bashrc /u/robot/.bashrc
-    sed -i -e "s/myrobot/${robot_name}/g" ~/.bashrc
-    sed -i -e "s/mydistro/indigo/g" ~/.bashrc #only working for indigo!!!
+    /u/robot/git/setup_cob4/cob-adduser robot
```

Comment:
does not make more sense unify the functions `SynchronizeRobotUser` and `SetupRobotBashrcWorkspace` if both call the cob-adduser script?

Reply:
in principal yes. but I'd like to extend the SetupRobotWorkspace with setting up workspace with wstool and rosinstall file. but not in this PR...